packages/preferences/src/browser/folders-preferences-provider.ts

```diff
@@ -113,17 +139,19 @@ export class FoldersPreferencesProvider extends PreferenceProvider {
         return provider;
     }
 
-    protected createFolderPreferenceProvider(folder: FileStat): FolderPreferenceProvider {
-        const provider = this.folderPreferenceProviderFactory({ folder });
+    protected createFolderPreferenceProvider(folder: FileStat, fileName: string): FolderPreferenceProvider {
```

Comment:
new args are breaking changes please update CHANGELOG for 0.6.0 with a record in breaking changes

Reply:
done